{"name": "test_login_failure", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1' == 1", "trace": "self = <test_feature_story.TestLogin object at 0x7ffa409d5b38>\n\n    @allure.story(\"密码缺失\")\n    def test_login_failure(self):\n        with allure.step(\"点击用户名\"):\n            print(\"输入用户名\")\n        with allure.step(\"点击密码\"):\n            print(\"输入密码\")\n        print(\"点击登录\")\n        with allure.step(\"点击登录之后登录失败\"):\n>           assert '1' == 1\nE           AssertionError: assert '1' == 1\n\ntest_feature_story.py:91: AssertionError"}, "steps": [{"name": "点击用户名", "status": "passed", "start": 1685431255533, "stop": 1685431255533}, {"name": "点击密码", "status": "passed", "start": 1685431255533, "stop": 1685431255533}, {"name": "点击登录之后登录失败", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1' == 1\n", "trace": "  File \"/Users/liaoboyang/pythonProject10测试报告/test_feature_story.py\", line 91, in test_login_failure\n    assert '1' == 1\n"}, "start": 1685431255533, "stop": 1685431255534}], "attachments": [{"name": "stdout", "source": "8be6af3b-4795-42c3-b40e-e6f39a134a86-attachment.txt", "type": "text/plain"}], "start": 1685431255533, "stop": 1685431255534, "uuid": "9afbf37a-8c8a-4eaa-9cf9-69c40eaed101", "historyId": "a34b019e1682c8c99a5a404099cd759a", "testCaseId": "a34b019e1682c8c99a5a404099cd759a", "fullName": "test_feature_story#test_login_failure", "labels": [{"name": "story", "value": "密码缺失"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_feature_story"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "liaoboyangdeMacBook-Air.local"}, {"name": "thread", "value": "13030-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_feature_story"}]}